lib/get/get-outdated-destination-content.js

```diff
@@ -36,7 +37,10 @@ Full error details below.
 
 function batchedIdQuery (space, method, ids) {
   return Promise.reduce(getIdBatches(ids), (fullResponse, batch) => {
-    return space[method]({'sys.id[in]': batch})
+    return space[method]({
+      'sys.id[in]': batch,
+      limit: batch.length
```

Comment:
`batch` is a string to get the correct limit it should be `limit: batch.split(',').length`